app/views/content/life-as-a-teacher/_categories.html.erb

```diff
@@ -38,6 +38,15 @@
               image: "static/images/content/hero-images/0030.jpg"
             )) %>
         </ul>
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Change to a career in teaching",
+              description: "Find out how to bring your skills and experience from a previous career to life in the classroom.",
+              path: "/life-as-a-teacher/change-careers",
+              image: "static/images/content/hero-images/student.jpg"
```

Comment:
I think this is a lovely image but I wonder whether an image with a teacher in it would be better here as it links better with the topic?

(Would be worth asking if the new batch of images will include any older teachers)

Reply:
Yeah I added that image as all the other pictures I could find were young teachers! Who's the contact for images?